docs/visualization/path/response_time.md

```diff
@@ -45,113 +45,113 @@ plot.show()
     The `output_notebook()` is needed to show figures inside Jupyter Notebook.
 <prettier-ignore-end>
 
-![response_time_all_histogram](../imgs/response_time_histogram_all.png)
+![response_time_histogram_all](../imgs/response_time_histogram_all.png)
 
 ```python
 # plot best case
 plot = Plot.create_response_time_histogram_plot(path, case='best')
 plot.show()
 ```
 
-![response_time_best_histogram](../imgs/response_time_histogram_best.png)
+![response_time_histogram_best](../imgs/response_time_histogram_best.png)
 
 ```python
 # plot worst case
 plot = Plot.create_response_time_histogram_plot(path, case='worst')
 plot.show()
 ```
 
-![response_time_worst_histogram](../imgs/response_time_histogram_worst.png)
+![response_time_histogram_worst](../imgs/response_time_histogram_worst.png)
 
 ```python
-# plot worst-in-input case
-plot = Plot.create_response_time_histogram_plot(path, case='worst-in-input')
+# plot worst-with-external-latency case
+plot = Plot.create_response_time_histogram_plot(path, case='worst-with-external-latency')
 plot.show()
 ```
 
-![response_time_worst_in_input_histogram](../imgs/response_time_histogram_worst_in_input.png)
+![response_time_histogram_worst_with_external_latency](../imgs/response_time_histogram_worst_with_external_latency.png)
 
 ## Stacked Bar
 
 The following scripts generate stacked bar graphs of response time.
-Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the worst, best, all, worst-in-input case.
+Horizontal axis of the stacked bar graph means timeseries and vertical axis means `Response time` in the all, best, worst, worst-with-external-latency case.
```

Comment:
In the sample graph, `latency` is displayed, but is it the same value as response time?
```suggestion
Horizontal axis of the stacked bar graph means `system time` or `index` and vertical axis means response time.
```

Reply:
Fixed it in [2211093](https://github.com/tier4/CARET_doc/pull/245/commits/2211093ddd6a8c037203df4f4072a5374e4bc7ab).

Is the description of the vertical axis of the stacked bar correct as per the change?